FirebaseMLModelDownloader.podspec

```diff
@@ -32,6 +32,7 @@ Pod::Spec.new do |s|
 
   s.framework = 'Foundation'
   s.dependency 'FirebaseCore', '~> 6.10'
+  s.dependency 'FirebaseInstallations', '~> 1.7'
```

Comment:
I recommend merging or rebasing the ml-model-downloader branch to master and Firebase 7.x dependencies separately from this PR.

Reply:
Makes sense. I opened #6891 for this.